pyaedt/application/Analysis3DLayout.py

```diff
@@ -76,20 +119,25 @@ def get_all_sparameter_list(self, excitation_names=[]):
 
     @aedt_exception_handler
     def get_all_return_loss_list(self, excitation_names=[], excitation_name_prefix=''):
-        """Get the list of all the Returnloss from a list of exctitations. If no excitation is provided it will provide a full list of return Losses
-        Example: excitation_names ["1","2"] output ["S(1,1)",, S(2,2)]
+        """Retrieve a list of all return losses for a list of excitations. 
 
         Parameters
         ----------
-        excitation_names :
-            list of excitation to include (Default value = [])
-        excitation_name_prefix :
-             (Default value = '')
+        excitation_names : list, optional
+            List of excitations. The default is ``[]``, in which case
+            the return losses for all excitations are to be provided. 
+        excitation_name_prefix : string, optional
+             Prefix to add to the excitation names. The default is ``""``.
 
         Returns
         -------
         type
-            list of string representing Return Losses of excitations
+            List of strings representing the return losses of the excitations.
+            
+        Examples
```

Comment:
I'm hoping that what was marked as "Example:" was meant to be placed in an Examples section because this is what I did throughout.

Reply:
Please see my previous suggestion and implement that for this and the other sections.  Really, it's going to have to take a developer to write the examples.  That's perhaps phase two or three of the documentation.